cmd/kind/get/clusters/clusters.go

```diff
@@ -41,12 +45,29 @@ func NewCommand() *cobra.Command {
 }
 
 func runE() error {
-	clusters, err := cluster.List()
+	clusters, err := nodes.ListByCluster()
 	if err != nil {
 		return err
 	}
-	for _, cluster := range clusters {
-		fmt.Println(cluster.Name())
+	printer := util.GetNewTabWriter(os.Stdout)
+	fmt.Fprintln(printer, strings.Join([]string{"NAME", "RUNNING", "VERSION"}, "\t"))
+	for cluster, nodeList := range clusters {
+		size := strconv.Itoa(len(nodeList))
+		running := 0
+		version := ""
+		for _, n := range nodeList {
+			status, _ := n.IsRunning()
```

Comment:
I don't know if we shouldn't show the nodes that belongs to the cluster but are not running

Reply:
I think it should be shown because containers are down unexpectedly, but currently we cannot know if each cluster's all containers are up easily. That's one of the reason of this PR.

Reply:
Ups, sorry, my bad, I didn't get it until now, you show the total and how many are running, that's great